google/cloud/bigquerycontrol/quickstart/BUILD.bazel

```diff
@@ -20,6 +20,6 @@ cc_binary(
         "quickstart.cc",
     ],
     deps = [
-        "@google_cloud_cpp//:experimental-bigquerycontrol",
```

Comment:
I don't think we should be changing the quickstart targets just yet

https://github.com/googleapis/google-cloud-cpp/blob/main/doc/contributor/howto-guide-declare-a-library-ga.md#reference-the-ga-targets-in-the-quickstarts

Reply:
Reverted.
<!-- Reviewable comment -ODqipIJA6bWwBN6PPXW:bp3aow6 -->
<!-- Sent from Reviewable.io -->